catalog/transformer/catalog-transformer-pdf/src/test/java/ddf/catalog/transformer/input/pdf/TestPdfInputTransformer.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.transformer.input.pdf;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.pdmodel.PDDocumentInformation;
+import org.junit.Before;
+import org.junit.Test;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.MetacardTypeImpl;
+import ddf.catalog.data.types.Contact;
+import ddf.catalog.data.types.Topic;
+import ddf.catalog.transform.CatalogTransformerException;
+
+public class TestPdfInputTransformer {
+
+    private PdfInputTransformer pdfInputTransformer;
+
+    private PDDocument pdDocument;
+
+    private PDDocumentInformation documentInformation;
+
+    @Before
+    public void setup() {
+        pdfInputTransformer = new PdfInputTransformer(mock(MetacardTypeImpl.class), false);
+
+        pdDocument = mock(PDDocument.class);
+        documentInformation = mock(PDDocumentInformation.class);
+
+        when(pdDocument.getDocumentInformation()).thenReturn(documentInformation);
+
+        pdfInputTransformer.setInputStreamPDDocumentFunction(inputStream -> pdDocument);
+        pdfInputTransformer.setThumbnailFunction(pdDocument1 -> new byte[0]);
+        pdfInputTransformer.setGeoParserFunction(pdDocument1 -> null);
+
+    }
+
+    @Test
+    public void testIsMetadataTitleTheMetacardTitleFalse() {
```

Comment:
Instead of verifying that `isUsePdfTitleAsTitle()` returns the value passed to `setUsePdfTitleAsTitle`, shouldn't we be verifying whether the title is being used or not based on the flag?